src/test/java/entities/validity_checkers/ValidityCheckerFacadeTests.java

```diff
@@ -0,0 +1,157 @@
+package entities.validity_checkers;
+
+import org.junit.After;
+import org.junit.Before;
+import org.junit.Test;
+
+import static org.junit.Assert.assertEquals;
+
+public class ValidityCheckerFacadeTests {
+
+    /**
+     * We create a custom implementation of ValidityCheckerFacade to have isValid return
+     * the punctuation and word with a space in between, so we can isolate them while testing
+     */
+    public static class TestValidityCheckerFacade extends ValidityCheckerFacade {
```

Comment:
ValidityCheckerFacade is already an implemented class with an is valid method. Since you are testing that class specifically, you want to create an instance of the class and pass in the dummy test checkers. Here you are extending the class and overriding the core method functionality, which means these tests are not testing the real Validity checker. There is no need for this class, simply make an instance of the existing ValidityCheckerFacade